ecommerce/templates/base_generic.html

```diff
@@ -13,6 +13,9 @@
 <body>
     {% include 'navbar.html' %}
     {% block content %}{% endblock %}
+    {% block jscode %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
line này chuyển ra load ở head phía trên nha em
```suggestion
    <script src="https://code.jquery.com/jquery-3.3.1.min.js" defer></script>
```